gradle/libs.versions.toml

```diff
@@ -6,8 +6,8 @@ androidxTestExt = "1.1.4-alpha03"
 androidxTestServices = "1.4.1"
 appCompat = "1.4.0"
 assertk = "0.25"
-coil = "2.0.0-alpha05"
-compose = "1.1.0-rc02"
+coil = "2.0.0-alpha06"
+compose = "1.1.0-rc01"
```

Comment:
@hvisser looks like this may be a bug — not sure why it wants to downgrade this, maybe because not all Compose dependencies have an RC02 version? (haven't checked, but it builds just fine right now)

Reply:
@rock3r I think there's an issue with the declaration in this case, only `androidx-compose-compiler` is at the `1.1.0-rc02` version, but the build file is referencing `libs.versions.compose`. All other libs using that version are not available in `rc02` so it seems that for that reason the downgrade happens as they show up in the dependency report as `exceeded`. If you do a build scan it will also tell you these dependencies failed to resolve, I guess other transitive dependencies pull the rc01 version in again or something. 

So to fix, I think the app build file should reference a separate version for the compiler, the rest seems good to me. Note that if you do this and the version isn't used elsewhere the plugin will remove it next time unless you set `keepUnused` to `true` (there's no way to detect you are using that version through in that manner I'm afraid).